src/main/resources/roleconfig/role_iac.json

```diff
@@ -901,5 +901,467 @@
         }
       }
     ]
+  },
+  {
+    "name": "specific-access-judiciary",
+    "label": "Specific Access (judiciary)",
+    "description": "Specific access for judicial users.",
+    "category": "JUDICIAL",
+    "substantive": false,
+    "patterns": [
+      {
+        "roleType": {
+          "mandatory": true,
+          "values": ["CASE"]
+        },
+        "grantType": {
+          "mandatory": true,
+          "values": ["BASIC"]
+        },
+        "classification": {
+          "mandatory": true,
+          "values": ["PRIVATE"]
+        },
+        "attributes": {
+          "jurisdiction": {
+            "mandatory": true
+          },
+          "caseType": {
+            "mandatory": true
+          },
+          "caseId": {
+            "mandatory": true
+          }
+        }
+      }
+    ]
+  },
+  {
+    "name": "specific-access-requested",
+    "label": "Specific Access Requested",
+    "description": "Specific access Requested for Judicial user.",
+    "category": "JUDICIAL",
+    "substantive": false,
+    "patterns": [
+      {
+        "roleType": {
+          "mandatory": true,
+          "values": ["CASE"]
+        },
+        "grantType": {
+          "mandatory": true,
+          "values": ["BASIC"]
+        },
+        "classification": {
+          "mandatory": true,
+          "values": ["PRIVATE"]
+        },
```

Comment:
For requested we need to retain grant type as SPECIFIC and classification as PRIVATE. for all the categories.